app/jobs/create_uphold_channel_card_job.rb

```diff
@@ -57,12 +60,15 @@ def find_or_create_card(uphold_connection, channel)
     [card_id, upfc]
   end
 
+  def card_exists?(uphold_connection, card_id)
+    return if card_id.blank?
+
+    uphold_connection.uphold_client.card.find(uphold_connection: uphold_connection, id: card_id).present?
+  rescue Faraday::ResourceNotFound
```

Comment:
I think it should return `false`

Reply:
https://github.com/brave-intl/publishers/pull/2200/commits/80a3639f7c23cf850d130406ef3db91bc8eb733d